semanticscholar/ApiRequester.py

```diff
@@ -1,25 +1,22 @@
-import json
 from typing import List, Union
 
-import requests
+import httpx
 from tenacity import (retry, retry_if_exception_type, stop_after_attempt,
                       wait_fixed)
 
 from semanticscholar.SemanticScholarException import \
-    BadQueryParametersException, ObjectNotFoundException
+    BadQueryParametersException, ObjectNotFoundException, \
+    TimeoutException
 
 
-class ApiRequester:
-    '''
-    This class handles calls to Semantic Scholar API.
-    '''
+class Requester:
```

Comment:
I think it's better to change names only when necessary to minimize the number of breaking changes.